spec/services/omni_auth_service_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe OmniAuthService do
+  let(:github_auth) { github_auth_hash }
+  let(:google_auth) { google_auth_hash }
+
+  describe '#user for github' do
```

Comment:
Could you add more conditions to this test? Some uncovered cases:
1) New identity, existing user
2) New identity, new user
3) Existing identity
4) And so on...